src/Notifier.ts

```diff
@@ -120,7 +121,19 @@ export const Notifier = {
             avatarUrl = Avatar.avatarUrlForMember(ev.sender, 40, 40, 'crop');
         }
 
-        const notif = plaf.displayNotification(title, msg, avatarUrl, room);
+        const content = ev.getContent();
+        let notif: Notification;
+        if (content.body !== undefined) {
+            let formattedBody = typeof content.formatted_body === 'string' ? content.formatted_body : null;
+            if (content.body.startsWith('>') && formattedBody) {
+                formattedBody = ReplyThread.stripHTMLReply(formattedBody);
+            }
```

Comment:
we have reply stripping code for things like message previews: breaking that out to a utility (if its not already) and using that would be appreciated, as this can mean a false positive as it currently stands.

Reply:
@turt2live 
Hi travis, 
Can you please tell me where is this reply stripping utility you talked about is located,
Thanks

Reply:
@turt2live 
Hey Travis,
I've moved the reply stripping code to utils.ts file.
Is that okay?

Reply:
> @turt2live
> Hey Travis,
> I've moved the reply stripping code to utils.ts file.
> Is that okay?

@turt2live 
Hey travis. Can you please let me know about this